src/main/java/connectify/logic/commands/SharePersonCommand.java

```diff
@@ -0,0 +1,115 @@
+package connectify.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import connectify.commons.core.index.Index;
+import connectify.commons.util.ToStringBuilder;
+import connectify.logic.Messages;
+import connectify.logic.commands.exceptions.CommandException;
+import connectify.model.Model;
+import connectify.model.person.Person;
+
+/**
+ * Shares a person to another address book.
+ */
+public class SharePersonCommand extends Command {
+
+    public static final String COMMAND_WORD = "sharePerson";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Shares instructions on how to add a Person to another address book.\n"
+            + "Parameters: "
+            + "INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_SHARE_PERSON_SUCCESS = "Command to add this Person:\n%1$s\n"
+            + "Do take note that you need to specify Company and priority on your own.";
+
+    private final Index targetIndex;
+
+    /**
+     * Creates a ShareCompanyCommand to share the specified {@code Company}
+     */
+    public SharePersonCommand(Index targetIndex) {
+        requireNonNull(targetIndex);
+        this.targetIndex = targetIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
```

Comment:
For SharePersonCommand, could you explore adjusting the command to request the user to specify both the company index as well as the person index within the company? The rationale is that the current version of Connectify specifies that every Person should be contained within a Company. You may reference the current "CompanyNoteCommand" and the current "DeleteCompanyCommand" for reference.

Reply:
I have resolved the SharePersonCommand syntax!